ivadomed/scripts/prepare_dataset_vertebral_labeling.py

```diff
@@ -41,6 +41,23 @@ def mask2label(path_label, aim=0):
     return list_label_image
 
```

Comment:
The file `ivadomed/scripts/prepare_dataset_vertebral_labeling.py` has had its permissions changed. (You can see this in the ["Files changed"](https://github.com/ivadomed/ivadomed/pull/787/files) view of your PR.)

![Screenshot from 2021-07-02 11-18-43](https://user-images.githubusercontent.com/16181459/124295830-4a331000-db27-11eb-8f94-f692d43ded11.png)


Please modify this file so that it will have the same `100644` permissions as it had before:

```
chmod 644 requirements.txt
```

Then commit the changes.

Reply:
Done